api/src/config.ts

```diff
@@ -51,8 +51,8 @@ export const config = {
       host: env.TR_DB_HOST || '127.0.0.1',
       port: parseInt(env.TR_DB_PORT, 10) || 3306,
       username: env.TR_DB_USER || 'root',
-      password: env.TR_DB_PASSWORD || '',
-      database: env.TR_DB_DATABASE || 'tr_dev',
+      password: env.TR_DB_PASSWORD || 'root',
+      database: env.TR_DB_DATABASE || 'ever-traduora',
```

Comment:
@rahul-rathore-576 let's keep old default name for DB for now, for backward compatibility for users who don't submit own DB name etc. I.e. revert changes for both default password and DB name please

Reply:
@evereq I have reverted my changes.